test/e2e_node/eviction_test.go

```diff
@@ -613,7 +617,11 @@ func runEvictionTest(f *framework.Framework, pressureTimeout time.Duration, expe
 				}
 				logFunc(ctx)
 				logKubeletLatencyMetrics(ctx, kubeletmetrics.EvictionStatsAgeKey)
-				return verifyEvictionOrdering(ctx, f, testSpecs)
+				err := verifyEvictionOrdering(ctx, f, testSpecs)
+				if err != nil {
+					framework.Logf("Verify eviction ordering failed: %s", err)
+				}
```

Comment:
Would it make sense to call `framework.Logf` in the `verifyEvictionOrdering` to avoid code duplication?

Reply:
Functions like `verifyEvictionOrdering` should not log errors. It's the decision of the caller how to handle an error (ignore it, check it, log it, etc,).

But I am wondering why this is needed. Gomega should log the last error when a timeout occurs, which is better than logging it during each poll.

According to the log:
```
There is no failure as the matcher passed to Consistently has not yet failed
```

So in other words, there hasn't been any error yet when the suite times out. Adding the log call above wouldn't help. Either the suite isn't allowed to run long enough (less likely?) or `verifyEvictionOrdering` gets stuck.

Reply:
It sounds like that this change is not valid.
/close

The eviction test failure is a time-out here.  Should we add more time here or not sure how to move this on?